lib/Mage/HTTP/Client/Curl.php

```diff
@@ -456,10 +454,34 @@ protected function parseHeaders($ch, $data)
         }
         $this->_headerCount++;
 
-
         return strlen($data);
     }
 
+    /**
+     * @param array $line
+     *
+     * @throws Exception
+     */
+    protected function validateHttpVersion(array $line)
+    {
+        if ($line[0] === 'HTTP/1.1') {
```

Comment:
What about http 1.0? Should it be also handled here?

Reply:
```suggestion
        if ($line[0] === 'HTTP/1.0' || $line[0] === 'HTTP/1.1') {
```